include/ebpf_nethooks.h

```diff
@@ -36,6 +36,26 @@ typedef enum _xdp_action
 typedef xdp_action_t
 xdp_hook_t(xdp_md_t* context);
 
+// XDP helper functions.
+#define XDP_EXT_HELPER_FN_BASE 0xFFFF
+
+#ifndef __doxygen
+#define EBPF_HELPER(return_type, name, args) typedef return_type(*name##_t) args
+#endif
+
+/**
+ * @brief Adjustss XDP context data pointer.
+ *
+ * @param[in] ctx XDP Context.
+ * @param[in] delta Number of bytes to move the data pointer by.
+ *
+ * @retval 0 on success, or a negative error in case of failure.
```

Comment:
retval generates a table where the first token is in the first column and the rest is in the second column, so this isn't formatted right
```suggestion
 * @retval 0 The operation was successful.
 * @retval <0 A failure occurred.
```